plugins/filter_cgolib/cgolib.c

```diff
@@ -0,0 +1,272 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2015-2018 Treasure Data Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include <fluent-bit/flb_filter.h>
+#include <fluent-bit/flb_utils.h>
+#include <fluent-bit/flb_pack.h>
+#include <fluent-bit/flb_parser.h>
+#include <fluent-bit/flb_unescape.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <dlfcn.h>
```

Comment:
Could you use [flb_dlfcn_win32.h](https://github.com/fluent/fluent-bit/blob/master/include/fluent-bit/flb_dlfcn_win32.h) on Windows?
This emulates dlfcn library on Windows:
https://github.com/fluent/fluent-bit/blob/master/src/flb_dlfcn_win32.c

Reply:
ok, I have fix it.